### DISCUSSION TITLE
Suggestion: Transpiling to Dart?

### DISCUSSION THREAD

**Discussion by novusnota** [pfusik/cito#163] [Ideas]
It's somewhat similar to Java and C# syntactically, so that aspect shouldn't be much difficult. Also, it has many semantical ideas from modern C#, like sound null safety.

More about the language: https://dart.dev
It's most commonly used for https://flutter.dev, although it's very usable on its own and provides an awesome developer tools and experience (IMHO).

**Comment by pfusik** [comment]
Good idea, although I have no resources to implement it any soon. Promoted to issue #164 anyway. :)
